maat-court-data-api/src/main/java/gov/uk/courtdata/repository/HardshipReviewRepository.java

```diff
@@ -15,4 +18,8 @@ public interface HardshipReviewRepository extends JpaRepository<HardshipReviewEn
 
     @Query(value = "SELECT hr FROM HardshipReviewEntity hr WHERE hr.repId = :repId AND hr.replaced = 'N'")
     HardshipReviewEntity findByRepId(@Param("repId") Integer repId);
+
+    @Query(value = "SELECT hr.* FROM TOGDATA.HARDSHIP_REVIEWS hr, TOGDATA.HARDSHIP_REVIEW_DETAILS hrd " +
+            "WHERE HR.ID = HRD.HARD_ID and HRD.HRDT_TYPE = :detailType AND HR.REP_ID = :repId", nativeQuery = true)
+    Optional<List<HardshipReviewEntity>> findByDetailType(@Param("detailType") String detailType, @Param("repId") Integer repId);
```

Comment:
Apologies, that signature isn't quite right!

Instead of `Optional<List<HardshipReviewEntity>>`
the return type should be `List<HardshipReviewEntity>`

As it wouldn't make sense to return an empty Optional if no records are returned, instead it should be an empty List.

Reply:
Updated the method signature.